package.json

```diff
@@ -28,9 +28,14 @@
     "styled-components": "^4.1.0"
   },
   "dependencies": {
+    "@types/react-slick": "^0.23.2",
```

Comment:
Can you move the rest of the types packages to `devDependencies` too?

Reply:
Sorry, should have seen that! Updated.